view/src/components/common/EventParticpantCard/EventParticpantCard.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import Paper from '@mui/material/Paper';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TablePagination from '@mui/material/TablePagination';
+import TableRow from '@mui/material/TableRow';
+import Box from '@mui/material/Box'
+import { useState, useEffect } from 'react';
+
+interface User {
+  id: number
+  name: string
+  number: number
+  created_at: number
+  updated_at: number
+};
+
+interface Events {
+  id: number
+  name: string
+  description: string
+  max_attendee: number
+  start_at: number
+  end_at: number
+  created_at: number
+  updated_at: number
+  user: User[]
+};
+
+const EventParticpantCard = () => {
+  const [eventusers, setEventUsers] = useState<Events[]>()
+
+  useEffect(() => {
+    fetch(process.env.CSR_API_URI + '/events/7/users', {
+      method: 'GET',})
+    .then(res => res.json())
+    .then(data => {
+        setEventUsers(data)
+        console.log(data)
+    })
+    .catch(error => {
+      console.error('エラーが発生しました', error);
+    });
+  },[])
+
+  const [page, setPage] = React.useState(0);
+  const [rowsPerPage, setRowsPerPage] = React.useState(10);
+
+  const handleChangePage = (event: unknown, newPage: number) => {
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setRowsPerPage(+event.target.value);
```

Comment:
+でNumberに型推論されるの初めて知った
参考：https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Operators/Unary_plus

すごいけどわかりにくいので、Number関数でラップして型変換しましょう！

```suggestion
    setRowsPerPage(Number(event.target.value));
```